Merged work on the embed share state and overall refactoring / cleanup of embed css

coryshaw opened the pull request:
- Share State Styles
- Refactored embed to be more dry
- code cleanup of styles on controls.css

coryshaw committed:
Added styles for share state, Refactored embed.less to be more DRY, cleanup

- Share State Styles
- Refactored embed to be more dry
- code cleanup of styles on controls.css

k88hudson commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f14bf4db83ae1ac/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f14bf4db83ae1ac/output.txt

Total script time: 1.09 mins

Server is running on http://pr844.butyr.org/templates/basic/
Run the Test Suite: http://pr844.butyr.org/test/

k88hudson commented on css/embed.less:
```diff
@@ -181,21 +181,41 @@ body {
-#post-roll {
   font-family: "Meta", "Open Sans", "Helvetica Neue", sans-serif;
-}
+  text-shadow: 1px 1px 0 rgba(255,255,255,0.4);
```
Our style is 
`rgba( 255, 255, 255, 0.4 );`

coryshaw replied:
You're referring to the spacing? Is that just a Moz standard or are there other benefits to that spacing?

k88hudson commented on css/embed.less:
```diff
@@ -181,21 +181,41 @@ body {
     position: relative;
     padding: 10%;
+    font-family: "Meta", "Open Sans", "Helvetica Neue", sans-serif;
+    text-shadow: 1px 1px 0px rgba(255,255,255,0.4);
```
same thing here

k88hudson commented on css/embed.less:
```diff
@@ -304,85 +320,82 @@ body {
+  
+  .share-options label {
+    display: block;
+    padding-bottom: 0px;
```
No `px` on 0 values

coryshaw replied:
Bad habit I need to break!

k88hudson commented on css/embed.less:
```diff
@@ -394,42 +407,14 @@ body {
-      font-size: 16px;
-      margin: 0 0 5px 0;
+      height:255px;
     }
```
space after `:`

k88hudson commented on css/embed.less:
```diff
@@ -437,100 +422,159 @@ body {
+    .embed-logo {
+      width: 20%;
+      margin-left: 20px;
+      margin-top: 0px;
```
no `px` on 0 values

k88hudson commented on css/embed.less:
```diff
@@ -437,100 +422,159 @@ body {
+
     .embed-project {
-      padding-left: @embedLogoWidth + 30px;
+      margin-top: 0px;
```
same thing here

coryshaw committed:
small tweaks per code review

- removed "px" after "0"
- added moz standard spacing on rgba declarations

coryshaw committed:
New Size Dropdown

- removed select drop down
- added div select menu and styles
- tweaked the js to pull in the new DOM structure, but it's not working
yet

k88hudson commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e58b6f9e4e63b6e/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e58b6f9e4e63b6e/output.txt

Total script time: 0.74 mins

Server is running on http://pr844.butyr.org/templates/basic/
Run the Test Suite: http://pr844.butyr.org/test/

coryshaw committed:
added js void to size drop down links, fixed buildIFrameHTML() function

coryshaw committed:
Merged Master

coryshaw committed:
hooked up js for embed size dropdown

coryshaw committed:
changed js void to preventDefault
